src/typhoon_automator/automator.py

```diff
@@ -213,8 +223,48 @@ def save_scenarios(
     def run(
             self,
             use_vhil: bool = False):
+        """ Run all scenarios in the automation
+
+        :param bool use_vhil:
+        """
+        
         self._model.load_to_setup(use_vhil=use_vhil)
-        raise NotImplementedError()
+        
+        if self._orchestrator is None:
+          raise RuntimeError("Automation is not initialized")
+    
+        start_time = datetime.now()
+        self._logger.info(f"Starting scenario simulations at {start_time.strftime('%H:%M:%S, %m/%d/%Y')}")
+    
+        self._orchestrator.run_all_scenarios()
+    
+        stop_time = datetime.now()
+        self._logger.info(f"Ended scenario simulations at {stop_time.strftime('%H:%M:%S, %m/%d/%Y')}")
+ 
 
     def shutdown(self):
-        raise NotImplementedError()
+        """ Shut down and clean up
+        """
+        
+        self._logger.info(f"Shutting down automation")
+  
+        # Stop simulation if needed
+        try:
+            if (self._simulation is not None) and (self._simulation.is_simulation_running()):
+                self._simulation.stop_simulation()
+        except:
+            self._logger.critical("Failed to stop simulation")
```

Comment:
Similar to logging info, this can be a call to the 'log' method.  Different severity levels can optionally be specified after the message:
```self.log(f"This is critically bad!", level = logging.CRITICAL)```